programme-schedule.md

```diff
@@ -0,0 +1,56 @@
+# Week 1: Induction
+
```

Comment:
I don't think it's best to add this document here. There's some points here that are for internal only. I suggest dropping this or replace it with a base skeleton for now.